Index: pages/api/hello.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>// Next.js API route support: https://nextjs.org/docs/api-routes/introduction\r\nimport type { NextApiRequest, NextApiResponse } from \"next\";\r\n\r\ntype Data = {\r\n\tname: string;\r\n};\r\n\r\nexport default function handler(\r\n\treq: NextApiRequest,\r\n\tres: NextApiResponse<Data>,\r\n) {\r\n\tres.status(200).json({ name: \"John Doe\" });\r\n}\r\n
===================================================================
diff --git a/pages/api/hello.ts b/pages/api/hello.ts
--- a/pages/api/hello.ts	
+++ b/pages/api/hello.ts	
@@ -6,7 +6,6 @@
 };
 
 export default function handler(
-	req: NextApiRequest,
 	res: NextApiResponse<Data>,
 ) {
 	res.status(200).json({ name: "John Doe" });
